docs/content/concepts/authorization/authorizers.md

```diff
@@ -270,13 +282,21 @@ The webhook will receive JSON-marshalled `SubjectAccessReview` objects, that (co
       "group2"
     ],
     "extra": {
-      "authorization.kubernetes.io/cluster-name": ["root"]
+      "authorization.kcp.io/cluster-name": ["root"]
     }
   }
 }
 ```
 
-    The extra field will contain the logical cluster _name_ (e.g. o43u2gh528rtfg721rg92), not the human-readable path. Webhooks need to resolve the name to a path themselves if necessary.
+The extra field will contain the logical cluster _name_ (e.g. o43u2gh528rtfg721rg92), not the human-readable path. Webhooks need to resolve the name to a path themselves if necessary.
+
+!!! note
+    In the past the cluster name was provided in the `authorization.kubernetes.io/cluster-name` extra field instead of `authorization.kcp.io/cluster-name`.
```

Comment:
Can we have an issue create to track this? So we dont forget?

Reply:
There is one but it's in an internal backlog. I can create a new one in kcp-dev/kcp